src/help/guides/getting-started.md

```diff
@@ -39,7 +42,7 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
 ### AltStore Mail plug-in for Mac
 1. Click the "**Install Mail Plug-in**" from the AltServer menu.
 1. Enter your password to grant AltServer permission to install the plug-in.
-1. Restart Mail and Open Mail > Preferences.
+1. On the mac restart Mail and Open Mail > Preferences.
```

Comment:
Should just be "Restart the Mail app and open preferences (CMD+,) "

Reply:
Yes, it was like that before but I did not find it explicit it has to be on the mac. Technically it would be difficult but it could also be on iOS.

I don't know if I am the only one in this case but I wondered were had I to install it.

Reply:
Since the last steps were for mac, it should be logical that the context would follow through to the next step unless specified that the step is to be performed on device.